From 0056ad033559371f0ce04d68fffc30c92fb551e8 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms@verge.net.au>
Date: Fri, 17 Feb 2012 16:16:28 +0900
Subject: Include linux/dma-mapping.h in linux/dmaengine.h

This reverses the effect of b7f080cfe223b3b7424872639d153695615a9255 (net:
remove mm.h inclusion from netdevice.h) while leaving almost all of the
code changes in place, in particular the creation of linux/dma-diretion.h
which is a dependency of subsequent patches that will be backported.

The motivation for this it to unbreak multiple drivers.
This patch cam be reverted if backports to all relevant drivers
are made.

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/linux/dmaengine.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/dmaengine.h b/include/linux/dmaengine.h
index 8fbf40e..8bf1032 100644
--- a/include/linux/dmaengine.h
+++ b/include/linux/dmaengine.h
@@ -23,7 +23,7 @@
 
 #include <linux/device.h>
 #include <linux/uio.h>
-#include <linux/dma-direction.h>
+#include <linux/dma-mapping.h>
 
 struct scatterlist;
 
-- 
1.7.10.2.565.gbd578b5

